static/index.js

```diff
@@ -80,24 +77,6 @@
     require('ipc').sendChannel('window-command', 'window:loaded')
   }
 
-  function setupAtomHome () {
```

Comment:
I should have left a comment in here originally about this but I think this is still needed on Linux based on this comment: https://github.com/atom/atom/issues/5412#issuecomment-73119795

Maybe we can pass `atomHome` as a load setting and set it as an env var still which would remove the code duplication.